charts/velocity-operator/templates/deployment.yaml

```diff
@@ -70,6 +70,10 @@ spec:
           - name: AWS_SHARED_CREDENTIALS_FILE
             value: {{ include "aws.credentials.path" . }}
           {{- end }}
+          {{- if ((.Values.velocity).apiUrl) }}
```

Comment:
What's the reason for the surrounding braces?

Reply:
older users of this chart don't have this value.

When they will upgrade, without the surrounding braces, the upgrade will fail because this is a nil value dereference.

Our CI pipeline validates that (it does an upgrade)

adding braces tells helm verifies if it is nil. if nil it is false

Reply:
Thanks!